app/controllers/users/registrations_controller.rb

```diff
@@ -0,0 +1,83 @@
+class Users::RegistrationsController < Devise::RegistrationsController
+
+  def new
+    @user = User.new
+    render 'devise/registrations/new'
+  end
+
+  def create
+    @user = User.new(sign_up_params)
+    @user.save
+    redirect_to challenges_path
```

Comment:
Same here. If we want to change the redirect path, we can just override the `after_sign_up_path` method that is commented out in the bottom of this controller. Don't need to override the entire action just for this.